gensim/corpora/sharded_corpus.py

```diff
@@ -233,10 +233,10 @@ def __init__(self, output_prefix, corpus, dim=None,
         self.current_shard = None    # The current shard itself (numpy ndarray)
         self.current_shard_n = None  # Current shard is the current_shard_n-th
         self.current_offset = None   # The index into the dataset which
-                                     # corresponds to index 0 of current shard
+        # corresponds to index 0 of current shard
```

Comment:
Don't split the comment: all in one line (either same line as var assignment, or the preceding line, but not after).